bio/fasta/fasta.go

```diff
@@ -56,12 +55,6 @@ Keoni
 
 ******************************************************************************/
 
-var (
```

Comment:
these were used for testing.

Reply:
All of these were used in tests that are moving out of `fasta` and into `bio`, where those tests can test all packages at once using generic functions.